recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -0,0 +1,414 @@
+#!/bin/sh
+
+usage="
+PHYTEC Onboarding Tool for ESEC IoT Device Manager Platform
+For More information: https://osb-cc-esec.github.io
+
+Usage:  $(basename $0) [acceptcontract] [OPTION]
+
+Example:
+        $(basename $0)
+        $(basename $0) acceptcontract --newaccount=<your mailing address>
+        $(basename $0) acceptcontract --onboarding
+
+One of the following options can be selected at a time:
+  acceptcontract                            Accept the contract and no Interface
+  -n, --newaccount=<your mailing address>   New account and Onboarding the device
+                                            for ESEC IoT Device Manager Platform
+  -o, --onboarding                          Only Onboarding the device for ESEC
+                                            IoT Device Manager Platform
+  -h. --help                                This Help
+"
+
+INTERACTIVE=True
+ESECCONTRACT=False
+
+# Get CPU UID
+get_cpu_uid() {
+    val=$(cat /sys/devices/soc0/soc_uid)
+    echo "${val}"
+}
+
+tpm2pkcs11tool='pkcs11-tool --module /usr/lib/libtpm2_pkcs11.so.0'
+CONFIG_PATH="/mnt/config"
+AWSCONFIG_PATH="${CONFIG_PATH}/aws"
+AWSCONFIG_CERT="certs"
+AWSCONFIG_CERTPATH="${AWSCONFIG_PATH}/${AWSCONFIG_CERT}"
+AWSCONFIG_CONFPATH="${AWSCONFIG_PATH}/config"
+ESECCONFIG="esec.config"
+
+TPM_PIN=$(get_cpu_uid | head -c 7)
+
+calc_wt_size() {
+    # NOTE: it's tempting to redirect stderr to /dev/null, so supress error
+    # output from tput. However in this case, tput detects neither stdout or
+    # stderr is a tty and so only gives default 80, 24 values
+    WT_HEIGHT=18
+    WT_WIDTH=$(tput cols)
+
+    if [ -z "$WT_WIDTH" ] || [ "$WT_WIDTH" -lt 60 ]; then
+        WT_WIDTH=80
+    fi
+    if [ "$WT_WIDTH" -gt 178 ]; then
+        WT_WIDTH=120
+    fi
+    WT_MENU_HEIGHT=$(($WT_HEIGHT-7))
+}
+
+do_finish() {
+    exit 0
+}
```

Comment:
Resolve this function and just use `exit 0` where needed.

Reply:
is changed